resources/views/modals/confirmDisableAllModal.php

```diff
@@ -1,24 +1,31 @@
 <!-- Main popup container with the form -->
-<div class="message-popup" id="confirmDisableAllModal" style="display: none">
-    <!-- Close button in the top-right corner -->
-    <!-- <img src="https://img.icons8.com/?size=100&id=71200&format=png&color=1A1A1A" alt="Close" class="close-icon" id="closePopup"> -->
-    <a href="#" class="plain-action" id="closePopup" onclick="closeModal('confirmDisableAllModal')"><i class="las la-times"></i></a>
+<div id="confirmDisableAllModal" class="modal">
+    <div class="modal-content">
+    
+        <!-- Botón de cerrar -->
+        <span class="close-button" onclick="closeModal('confirmDisableAllModal')">
+          <i class="fas fa-times"></i>
+        </span>
+       <!-- Título de la ventana -->
+        <h2>
+            <i class="las la-lock"></i>
+            ¿Desea desactivar todas las cuentas?
+        </h2>
+        
+        <form action="/admin/disable/all/accounts" method="POST">
+    
+            <div class="form-group">
+                <p>Esta acción eliminará los documentos de las cuentas y cambiará el estado de las mismas a "desactivada".  Esta acción no se podrá revertir...</p>
+            </div>
 
-    <!-- Popup title -->
-    <h2 class="message-title">¿Desea desactivar todas las cuentas?</h2>
-
-    <!-- Alert -->
-    <div class="message-options">
-        <h3> Esta acción eliminará los documentos de las cuentas y cambiará el estado de las mismas a "desactivada".  Esta acción no se podrá revertir... </h3>
-    </div>
-
-
-    <!-- Buttons area -->
-    <div class="modal-actions">
-        <!-- Cancel button -->
-        <a href="#" class="secondary main-action-bright" onclick="closeModal('confirmDisableAllModal')">Cancelar</a>
-
-        <!-- Confirm button -->
-        <button class="primary main-action-bright" onclick="closeModal('confirmDisableAllModal')">Confirmar</button>
+            <div class="modal-actions">
+                <a class="main-action-bright" onclick="closeModal('confirmDisableAllModal')">Cancelar</a>
+                <button type="submit" class="main-action-bright danger">
+                    <i class="las la-lock"></i> 
```

Comment:
use font awesome icon
```suggestion
                    <i class="fas fa-lock"></i> 
```